fe/src/components/home/popup/ActivityInfoPopup.tsx

```
@@ -0,0 +1,88 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { Map, MapMarker, ZoomControl, Loader } from "react-kakao-maps-sdk";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)